lib/readline.js

```diff
@@ -110,8 +110,15 @@ const {
 
 const { StringDecoder } = require('string_decoder');
 
-// Lazy load Readable for startup performance.
-let Readable;
+const {
+  Readable,
+  Writable
```

Comment:
Isn't it possible to add some jsdoc comment that "imports" this type?

Reply:
I think so.